content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -0,0 +1,164 @@
+
+**Note**: MKE (Mirantis Kubernetes Engine) does not support the installation of CSI-PowerFlex via Operator.
+
+### Listing installed drivers with the ContainerStorageModule CRD
+User can query for all Dell CSI drivers using this command:
+`kubectl get csm --all-namespaces`
+
+### Prerequisites
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../../troubleshooting/powerflex.md) for details
+
+#### SDC Deployment for Operator 
+- This feature deploys the sdc kernel modules on all nodes with the help of an init container.
+- For non-supported versions of the OS also do the manual SDC deployment steps given below. Refer to https://hub.docker.com/r/dellemc/sdc for supported versions.
+- **Note:** When the driver is created, MDM value for initContainers in driver CR is set by the operator from mdm attributes in the driver configuration file, 
+  config.yaml. An example of config.yaml is below in this document. Do not set MDM value for initContainers in the driver CR file manually.
+- **Note:** To use an sdc-binary module from customer ftp site:
+  - Create a secret, sdc-repo-secret.yaml to contain the credentials for the private repo. To generate the base64 encoding of a credential:
+ ```yaml
+      echo -n <credential>| base64 -i
+``` 
+   secret sample to use:
+ ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: sdc-repo-creds
+        namespace: vxflexos
+      type: Opaque
+      data:
+        # set username to the base64 encoded username, sdc default is
+          username: <username in base64>
+        # set password to the base64 encoded password, sdc default is
+          password: <password in base64>
+```  
+  - Create secret for FTP side by using the command `kubectl create -f sdc-repo-secret.yaml`.
+  - Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note: 
+    - **If using sidecar**, you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. 
+    - **If not using sidecar**, please leave this commented out -- otherwise, the empty fields will cause errors.
+##### Example CR:  [samples/storage_csm_powerflex_v240.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerflex_v240.yaml)
+```yaml
+    sideCars:
+    # sdc-monitor is disabled by default, due to high CPU usage 
+      - name: sdc-monitor
+        enabled: false
+        image: dellemc/sdc:3.6
+        envs:
+        - name: HOST_PID
+          value: "1"
+        - name: MDM
+          value: "10.xx.xx.xx,10.xx.xx.xx" #provide MDM value
+```  
+ *Note:* Please comment the sdc-monitor sidecar section if you are not using it. Blank values for MDM will result in error.
```

Comment:
We should change this line. Now users can keep enabled to false to skip monitor install